tests/capture/test_capture_transforms.py

```diff
@@ -199,3 +202,255 @@ def g(i):
 
         assert qfunc_jaxpr.eqns[1].primitive == qml.Z._primitive
         assert qfunc_jaxpr.eqns[2].primitive == qml.measurements.ExpectationMP._obs_primitive
+
+
+# pylint: disable=protected-access, expression-not-assigned
+class TestMapWiresTransform:
```

Comment:
Before we get into the integration tests, can we have some unit tests?

Maybe even something just like:
```
@MapWires({0:1})
def f(x):
    return qml.RX(x, 0)

expected_out = qml.RX(0.5, 1)
qml.assert_equal(f(0.5), expected_out)

jaxpr = jax.make_jaxpr(f)(0.5)
...
```
where are just focusing in on one single operation?